card-flow.md

```diff
@@ -0,0 +1,52 @@
+# Project Management Boards
+
+Our team uses project boards to manage our projects and track our progress.  The boards serve two purposes: to allow clients to see what's happening and give feedback on a project, and to allow us to manage our team, our tasks, and stay coordinated. Each project has its own board. The board shows us the current state of all the things we're working on in our project and where each task is in the lifecycle of development. Each task is represented by a card on the board, and as cards move through the development process, we move them to different columns on the board.
+
+Everything the team is working on for a project, whether a feature, a bug, or a chore, should be represented by a card. We can't track something if it's not on the board. If we're doing something that's not represented by a card, we should ask why.
+
+## Board Flow
+
+Our current project board tool is not very expressive in some ways, and this leads us to add additional columns to our board. Currently, we have a "Backlog" column in our project board. Once Feature Complete is up and running, our backlog will live in the Features area of the app rather than on the project board itself. The "Backlog" column is where we store tasks that will need to be completed but it's either not the right time to begin work on them or they haven't yet been approved.
```

Comment:
Let's replace this section of the board flow with something like the following.  Feel free to edit/clean up.

## Board Flow

In Kanban, we add cards on the far left-most column that are to be worked on by the team. As cards are ready to be worked on and there's availability by the team, we move the cards to the right as it goes through the steps of software development.  Each column represents the stages that a specific project has defined.  We will illustrate our common flow below but each project may modify this based on business conditions.

### Where's the backlog or icebox columns?

Some tools have a place to keep cards that don't 'fit' into the current iteration or cycle of work.  This concept doesn't exist in Kanban and it's been our experience that trying to mimic either backlogs or iceboxes lead to out-dated and wasteful cards that ultimately slow down the team.  Instead we have the client keep any extra work not quite ready to be tackled elsewhere.  This allows only 'fresh' ideas to enter the board to be worked on by the team in the next 1-2 weeks at most.